### PR TITLE
fix: HMPB renders on page reload.

### PR DESCRIPTION
**Pull request by beausmith**

### CODE REVIEW

File: client/src/components/HandMarkedPaperBallot.tsx [hunk 1]
```diff
@@ -261,23 +261,17 @@ const HandMarkedPaperBallot = ({
...
-        ['/ballot/ballot.css'],
-        printBallot
-      )
-      console.log('preview rendered, total pages', flow.total, { flow })
-      onRendered?.({ ballotStyleId, election, isLiveMode, precinctId, votes })
-    })()
+    new Previewer().preview(
+      ballotRef.current!.innerHTML,
+      ['/ballot/ballot.css'],
+      printBallot
```
Comment (benadida) [inline client/src/components/HandMarkedPaperBallot.tsx:267]: so you're okay if this is null? Or maybe it never was?

File: client/src/components/HandMarkedPaperBallot.tsx [hunk 2]
```diff
@@ -261,23 +261,17 @@ const HandMarkedPaperBallot = ({
...
+    )
+    onRendered?.({ ballotStyleId, election, isLiveMode, precinctId, votes })
 
     return () => {
-      // console.log('removing pagedjs ballot')
-      printBallot.innerHTML = ''
+      if (!!printBallot) {
+        printBallot.innerHTML = ''
+      }
```
Comment (eventualbuddha) [inline client/src/components/HandMarkedPaperBallot.tsx:274]:
```suggestion
      printBallot?.innerHTML = ''
```

Comment (beausmith) [reply client/src/components/HandMarkedPaperBallot.tsx:274]: image

File: client/src/components/HandMarkedPaperBallot.tsx (hunk 1 above)
Comment (beausmith) [reply client/src/components/HandMarkedPaperBallot.tsx:267]: I've added some logging to try and understand what happens… I'm not sure how it's working, but it is. 🤷‍♂️ I'm worried that I don't understand it.

File: client/src/components/HandMarkedPaperBallot.tsx (hunk 2 above)
Comment (beausmith) [reply client/src/components/HandMarkedPaperBallot.tsx:274]: I also tried `printBallot && printBallot.innerHTML = ''` but then got a warning that there was a missing `;` somewhere 🤷

Comment (eventualbuddha) [reply client/src/components/HandMarkedPaperBallot.tsx:274]: Ah, right. Well at least you can just do `if (printBallot) {` rather than `!!`.

**Pull request merged**